Update installation_der_software.md

hnrstrck opened the pull request:
Link war veraltet, sodass die Lib nicht mit der aktuellen Arduino-Software kompatibel war. Eventuell würde es sich anbieten, eine mit Screenshots bebilderte Anleitung zur Installation der Lib in das Buch / die Bücher mit aufzunehmen.

Commit:
Update installation_der_software.md

Link war veraltet, sodass die Lib nicht mit der aktuellen Arduino-Software kompatibel war. Eventuell würde es sich anbieten, eine mit Screenshots bebilderte Anleitung zur Installation der Lib in das Buch / die Bücher mit aufzunehmen.

ubergesundheit commented:
Der Link ist so wie er ist schon korrekt. Die verlinkte Zip-Datei enthält die Beispiele und mehrere Libraries die benötigt werden.

Was genau ist denn nicht mehr auf dem neusten Stand? Dies müsste dann im senseBox/resources repository in der dort enthaltenen Zip Datei eingepflegt werden

hnrstrck commented:
Das Einbinden der Lib über das Menu der Arduino-IDE führt zu einem Fehler (habe den genauen Wortlaut gerade nicht vorliegen). Lädt man das Master des entsprechenden Repos als zip runter und bindet dieses ein, funktioniert es.

ubergesundheit commented:
Ok, der pull request bezieht sich auf die senseBox:edu, deswegen kann das fast schon sein. Für die senseBox:home wird das Funktionieren des Zip Archiv im repository [node-sketch-templater](https://github.com/sensebox/node-sketch-templater) per CI getestet.

Denke dann doch dass du Recht hast, dass die verlinkte Datei nicht für die Edu Beispiele funktioniert! 

Dann würde deine Änderungen aber nicht weit genug gehen, denn im nächsten Absatz wird von den von mir genannten Verzeichnissen `examples` und `libraries` gesprochen. Die in dem von dir verlinkten Archiv nicht enthalten sind.

Könntest du dann den Rest des Kapitels bitte auch noch soweit anpassen, dass es zu dem von dir verlinkten Archiv passt? Danke!

hnrstrck commented:
Ah, okay, das wusste ich nicht. Ich ändere das (ggfs. nächste Woche / zu Beginn der Osterferien).

> On 14. Mar 2018, at 14:09, Gerald Pape <notifications@github.com> wrote:
> 
> Ok, der pull request bezieht sich auf die senseBox:edu, deswegen kann das fast schon sein. Für die senseBox:home wird das Funktionieren des Zip Archiv im repository node-sketch-templater <https://github.com/sensebox/node-sketch-templater> per CI getestet.
> 
> Denke dann doch dass du Recht hast, dass die verlinkte Datei nicht für die Edu Beispiele funktioniert!
> 
> Dann würde deine Änderungen aber nicht weit genug gehen, denn im nächsten Absatz wird von den von mir genannten Verzeichnissen examples und libraries gesprochen. Die in dem von dir verlinkten Archiv nicht enthalten sind.
> 
> Könntest du dann den Rest des Kapitels bitte auch noch soweit anpassen, dass es zu dem von dir verlinkten Archiv passt? Danke!
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/sensebox/books/pull/51#issuecomment-373014935>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AVGpbWVwLNDXxZERY5hYzdjyFbW46T6rks5teRaigaJpZM4Some7>.
>

Commit:
Update installation_der_software.md

Vorschlag für Änderung der Bibliotheken-Installation über das Menü der Arduino-Software
